chat-components/src/components/prechatsurveypane/common/defaultProps/defaultStyles/defaultPreChatSurveyPaneButtonStyles.ts

```diff
@@ -2,5 +2,6 @@ import { IPreChatSurveyPaneButtonStyles } from "../../../interfaces/IPreChatSurv
 
 export const defaultPreChatSurveyPaneButtonStyles: IPreChatSurveyPaneButtonStyles = {
     backgroundColor: "rgb(49, 95, 162)",
-    color: "#FFFFFF"
+    color: "#FFFFFF",
+    fontFamily: "sans-serif"
```

Comment:
It looks like we use "Segoe UI, Arial, sans-serif" in most parts. Do we want to change to the same font family?